oraclepaas/resource_java_service_instance.go

```diff
@@ -789,6 +839,12 @@ func resourceOraclePAASJavaServiceInstanceRead(d *schema.ResourceData, meta inte
 		d.Set("assign_public_ip", val)
 	}
 
+	if result.LoadBalancer != nil {
+		if err := d.Set("load_balancer", flattenLoadBalancer(d, *result.LoadBalancer)); err != nil {
```

Comment:
by nil-checking this we're not allowing this to be removed; which whilst technically correct given this is ForceNew, in general it's better to allow flattenLoadBalancer to take a pointer and then return an empty item if it's nil imo

Reply:
Ah, that makes sense to me. Fixed up now